code/index.html

```diff
@@ -8,15 +10,109 @@
   </head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+
+      <!--the Hamburger icon (not yet interactive)-->
+      <div class ="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <!-- logo -->
+      <div class="logo">
+        <a href="#">
+          <img src="/assets/splurmf news logo 2.svg" alt="logo">
```

Comment:
Name all files with kebab-case: "splurmf-news-logo-2.svg". Paths can be tricky in the beginning. Since "assets" is a sibling folder to "code", you need to specify that it can be reached by going up one level and then down to assets. "../assets/splurmf-news-logo-2.svg" -> where "../" lands us in the "project-news-site" root folder. Hope that makes sense, otherwise [here](https://www.w3schools.com/html/html_filepaths.asp)'s some good reading on the subject.

Reply:
If you change all file paths accordingly 👆, your deployed site should display the images :)